package.json

```diff
@@ -7,6 +7,8 @@
     "dev": "nodemon --exec dotenv ts-node src/index.ts",
     "test": "NODE_ENV=test jest",
     "coverage": "yarn codecov",
+    "build": "tsc -p tsconfig.json",
+    "start": "node -r dotenv/config dist/index.js",
```

Comment:
We should not use dotenv in production environment. We expect the environment variables to be already there.

Reply:
done